### DISCUSSION TITLE
explain cardinals

### DISCUSSION THREAD

**Discussion by mikaba-eu** [ordinals/ord#2392] [Q&A]
Can someone explain what the cardinal amount is and why we are only able to add 5 ordinals and have then to wait until all transactions are succeeded? I can't find a good documentation about the cardinals.

**Comment by refazi** [comment]
Cardinals are the inputs for the ordinal tx that the wallet makes, in order to inscribe more at a time you should take a big input or multiple bit inputs and split it to cardinals with size that would cover the inscription fee (use --dry-run for estimates)

**Comment by mikaba-eu** [comment]
so basically I have to send X transactions with Y sats per transaction from my main wallet to the ord wallet there X is the amount of ords that I want create? And Y is determinated by the dry-run? Do I need multiple receive addresses for that?

**Comment by satoshi0770** [comment]
> so basically I have to send X transactions with Y sats per transaction from my main wallet to the ord wallet there X is the amount of ords that I want create? And Y is determinated by the dry-run? Do I need multiple receive addresses for that?

y + 10k sats for postage. but put a little more just incase the calc from --dry-run isnt exact.
you can send to the same receive address, but general best practice in bitcoin is not to re-use addresses.
you can construct a multi output tx where 1 input and sends to 10 outputs to fund 10 utxos to use for inscribing. 
(can do this with sparrow for an easier gui)

ord inscribe creates 2 transactions, a comit tx and a reveal tx.
and it only uses utxos that have been included in a block (already confirmed) for the comit.
so if wanting to inscribe multiple at once you need multiple utxos ready with cardinals. 
cardinals are sats that have no inscriptions. 

if need more help, jump in the ordicord (ordinals discord), as its much more active with many ppl ready to help.
utilize the search function, and use the #techsupport channel, or the voice chat.

hope that helps
